webserver/tests/sse/src/test/java/io/helidon/webserver/tests/sse/SseBaseTest.java

```diff
@@ -45,18 +45,28 @@ protected WebServer webServer() {
         return webServer;
     }
 
+    private static void sleep(long millis) {
+        try {
+            Thread.sleep(millis);
+        } catch (InterruptedException e) {
+            Thread.currentThread().interrupt();
+        }
+    }
+
     static void sseString1(ServerRequest req, ServerResponse res) {
         try (SseSink sseSink = res.sink(SseSink.TYPE)) {
             sseSink.emit(SseEvent.create("hello"))
                     .emit(SseEvent.create("world"));
+            // Fix for github actions: https://github.com/helidon-io/helidon/issues/10471
```

Comment:
Sleep is in general not a good solution to intermittent failures, as the amount of time required may change depending on the runner.
If we want to fix tests properly, we should have a mechanism to correctly chain the events (i.e. using some Java concurrent constructs, such as a CountDownLatch).

Reply:
Right, I will do that.

What about the flush in PlainSocket#close?.